doc/whats_new.rst

```diff
@@ -35,6 +35,7 @@ API
 ~~~
 
 - Add topographic feature using MNE with `~mne_icalabel.features.get_topomap_array` and `~mne_icalabel.features.get_topomaps` by `Anand Saini`_ and `Mathieu Scheltienne`_ (:gh:`71`)
+- Add psd feature using MNE with `~mne_icalabel.features.get_psds` by `Anand Saini`_ (:gh:`73`)
```

Comment:
```suggestion

- Add PSD feature using `mne.time_frequency.psd_multitaper` with `~mne_icalabel.features.get_psds` by `Anand Saini`_ (:gh:`73`)
```

Reply:
Entries in the changelog are separated by one empty line, else the documentation build will complain ;)